CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Changelog
 
+
```

Comment:
Notes from gabriel regarding frontend: 

- frontend: add support for Polygon zkEVM (mainnet still not enabled)
- frontend: tokens/chains visibility can now be controlled easily
- frontend: fix minor bugs
- frontend: add support for USDT & WETH
- frontend: migrate to WalletConnect v2
- frontend: users can now fully disconnect from the connected wallets (session removed)
- frontend: update wallet provider SDKs
- docker: fix flaky polygon e2e test
- relayer: prevent submission of failing L1 resolutions

Reply:
only changed USDT & WETH from frontend to beamer as we did changes on contracts and also agent config for that.